hlxsites/merative2
# feat(quick-links): Add POC LinkTree/Bitly template for showing quick links

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-821

## Description

**New**

- This PR adds a new template and blocks for showcasing quick links.
- This is a POC (proof of concept) to quickly leverage Franklin as a means to create helpful links pages that can be used by marketers in conference/event booths. Allowing customers/users to quickly scan the QR code to access helpful links within their mobile devices.
- **NOTE:** The design of this template is not finalized. I just put together a layout in code to showcase what the different products will look like in their own branding color and pattern.

## Design Specs

- Figma Link - No design specs



## Micromedex - Quick Links

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/micromedex
- After (Changes from this PR): https://feat-quicklinks-poc--merative2--proeung.hlx.page/quick-links/micromedex

![Screenshot 2023-08-25 at 2 51 51 PM](https://github.com/hlxsites/merative2/assets/1815714/00732789-5f90-4ba5-a497-18c53b7cf45e)

![Screenshot 2023-08-25 at 2 50 11 PM](https://github.com/hlxsites/merative2/assets/1815714/64b295d6-0434-4479-874b-147d353d7715)


## Health Insights - Quick Links

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/health-insights
- After (Changes from this PR): https://feat-quicklinks-poc--merative2--proeung.hlx.page/quick-links/health-insights

![Screenshot 2023-08-25 at 2 51 55 PM](https://github.com/hlxsites/merative2/assets/1815714/c122ade3-b96b-46ec-bbf3-802c7dedb87b)
![Screenshot 2023-08-25 at 2 50 15 PM](https://github.com/hlxsites/merative2/assets/1815714/72cfc486-5da3-4dfd-89cd-c28830777317)
  
## Testing Instruction

- Visit the preview links above and ensure that the CTA test links are functioning correctly.
